shared/src/commonMain/kotlin/org/hyperskill/app/step_quiz/presentation/StepQuizReducer.kt

```diff
@@ -213,7 +216,10 @@ class StepQuizReducer(private val stepRoute: StepRoute) : StateReducer<State, Me
                     val event = StepQuizDailyStepCompletedModalClickedGoBackHyperskillAnalyticEvent(
                         route = stepRoute.analyticRoute
                     )
-                    state to setOf(Action.LogAnalyticEvent(event), Action.ViewAction.NavigateTo.Back)
```

Comment:
OK, but why a lot of code reformatting if it successfully fits the line length of `120 symbols`